src/actions/API.actions.js

```diff
@@ -499,18 +499,22 @@ export function getTTSLangText(voiceList){
       dataType: 'json',
       crossDomain: true,
       timeout: 3000,
-      async: false,
+      async: true,
+
       success: function (response) {
         if(response[0]){
           if(response[0][0]){
             let translatedText = response[0][0][0];
             voice.translatedText = translatedText;
+            console.log(url);
```

Comment:
@tstreamDOTh  Why log the URL every time?
Is this something you were using for debugging? because my console looks like this
![image](https://user-images.githubusercontent.com/31416811/32408120-c77b9f32-c1b8-11e7-86ad-bc6593592cf1.png)

Reply:
Yes, it's for debugging purpose to ensure if all the ajax calls were made like how it is being done previously

Reply:
Should be fine to remove then I guess.